english/magicaladventure/avada_kedavra/avada_kedavra.md

```diff
@@ -0,0 +1,7 @@
+No, Luke ... I mean Harry ... don't choose the way to the dark I mean Death Eater side.
+
+Voldemort dies from your vengeful response, and your lust for revenge is satiated.
+
+- - - - - - 20 years later
+
+You're the Minister of Magic, having crushed the opposition and determine to let no one ever commit anything you call injustice. Dumbledore weeps in his grave over what you've become.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.